db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -1,21 +1,17 @@
+gem "net-ssh", "=4.0.0.alpha4"
```

Comment:
@alongoldboim I'd prefer to put this in the Gemfile while this is WIP and change it to 3.2.x when it's released.